File: exercism/go/hamming/hamming.go
package hamming

import "errors"

func Distance(a, b string) (int, error) {
    if len(a) != len(b) {return 0, errors.New("must be of same length")}
    counter := 0
    for i, v := range a {
        if byte(v) != b[i] {counter++}
    }
    return counter, nil
}

File: exercism/python/hamming/hamming.py
def distance(strand_a, strand_b):
    hamming = 0;
    if (len(strand_a) != len(strand_b)):
        raise Exception("Error, strands must be of same length")
        return hamming
    for a, b in zip(strand_a, strand_b):
        if (a != b): hamming += 1
    return hamming
